manifests/s/Seewo/EasiNote/5.2.4.5928/Seewo.EasiNote.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.12 by Dumplings $debug=QUSU.CRLF.7-2-17.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Seewo.EasiNote
+PackageVersion: 5.2.4.5928
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: install
+Protocols:
+- easinote
+FileExtensions:
+- enb
+- enbx
+- enpx
+ReleaseDate: 2024-01-05
+Installers:
+- Architecture: x86
+  InstallerUrl: https://cstore-pub-cos-seewo-report-tx.seewo.com/seewo-report_uwixvhimhhulnhlwhloummlwqjphihhh?sign=q-sign-algorithm%3Dsha1%26q-ak%3DAKIDJUXMxJRLzPaeMp20jDSTFl23pLcdPwDF%26q-sign-time%3D1704438416%3B2019798416%26q-key-time%3D1704438416%3B2019798416%26q-header-list%3Dhost%26q-url-param-list%3Dresponse-content-disposition%26q-signature%3D7fa4b4645242de0f5c76f653adfa65a16e85f91a&response-content-disposition=attachment%3Bfilename%3D%22EasiNoteSetup_5.2.4.5928_seewo.exe%22%3Bfilename%2A%3Dutf-8%27%27EasiNoteSetup_5.2.4.5928_seewo.exe

```

Comment:
This is a time-based URL, with a `q-sign-time` of 1704438416 (Friday, January 5, 2024 7:06:56 AM GMT) or about 11 hours ago.

Reply:
idk but ig it doesn't really matter as the first version of this package available in the repo still works.